### ISSUE TITLE
On the card dialog box, language of part is not defined.

### ISSUE THREAD

**Issue by azaslonov** [paperbits/paperbits-demo#132]
User experience:
If, the language of part is not defined for the content of dialog box, then the user who depends upon Screen Reader for navigation might get affected hence failed to proper information of the content available on the dialog box.

Note: User credentials should NOT be included in the bug.

Pre-requisites:

1. Open Narrator via pressing ctrl+window+enter key and press caps lock+spacebar to activate scan mode.
2. Open developer toolbar via pressing F12 key.

Repro steps:

1. Open Azure portal with valid credentials and create a API Management.
2. Click Developer Portal button to navigate to the portal.
3. Wait portal to load in design mode
4. Navigate to the Left nav and select Pages button.
5. Navigate to the Pages window. Select "APIs: List" option.
6. Navigate to the any editable control details and select Add widget.
7. Navigate to the Card and select it.
8. Navigate with Screen Reader to the various controls of the "Card" dialog box.
9. Open developer toolbar via pressing F12 key and check the language of part is defined for the dialog box content or not.

Actual Result:

On the card dialog box, language of part is not defined.
 

Expected Result:

On the card dialog box, language of part should be defined. So, that the Screen Reader user does able to get the proper information of the content available on the dialog box.

Note: This issue also exist on the Chrome browser.
 
MAS Reference:

MAS 3.1.2 - Language of Parts

Reference Links

Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.
External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it      back to the tester with notes on where to file/route the bug. For more information please use this Link. “HCL Staff should not log any third party external bugs. Should be routed to edad team”.
Please reach out to C&AI Teams channel for any process related queries.

![image](https://user-images.githubusercontent.com/2320302/79022497-ed6ac480-7b32-11ea-88ae-a106a450fb0f.png)

**Comment by azaslonov** [comment]
Language parts are not supported. The example content on the default Card template can be changed by a user.
